boards/arm/frdm_k64f/pinmux.c

```diff
@@ -166,6 +166,10 @@ static int frdm_k64f_pinmux_init(const struct device *dev)
 		       PORT_PCR_PE_MASK | PORT_PCR_PS_MASK);
 #endif
 
+#if CONFIG_SHIELD_ADAFRUIT_2_8_TFT_TOUCH_V2
+	pinmux_pin_set(portc,  4, PORT_PCR_MUX(kPORT_MuxAsGpio));
```

Comment:
I do not think it is right, you do not want to have explicit configuration for all possible shields here. Another way should be found.

Reply:
Agree here that an alternative approach should be found but maybe this is better done in a separate PR

Reply:
Agree but I'd like to postpone this until after we move NXP parts to use device tree pinmuxing.